app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :commments
+  has_many :likes
+  scope :posts, -> { joins(:posts).where(posts: { user_id: id }) }
+
+  def last_3_posts
+    posts.limit(3)
+  end
```

Comment:
- [ ] Even though this method would take only 3 posts from the user, the requirement says to create a method that takes the 3 `most recent` posts, so you should try using the `created_at` property from SQL probably instead of just limiting it to 3. Please work on this issue.